bindings/ruby/src/lib.rs

```diff
@@ -26,6 +26,7 @@ pub use ::opendal as ocore;
 
 mod capability;
 mod metadata;
+mod opendal_io;
```

Comment:
Having a module named `opendal_io` feels a bit odd to me. Could we simply call it `io` instead?

Reply:
Of course, thanks for your suggestion. Changes to `Io` following Rust naming convention.